src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderItemUnitRepository.php

```diff
@@ -13,17 +13,17 @@
 
 namespace Sylius\Bundle\CoreBundle\Doctrine\ORM;
 
-use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderItemUnitRepository as BaseOrderItemUnitRepository;
 use Sylius\Component\Core\Model\CustomerInterface;
 use Sylius\Component\Core\Model\OrderItemUnitInterface;
 use Sylius\Component\Core\Repository\OrderItemUnitRepositoryInterface;
 
 /**
  * @template T of OrderItemUnitInterface
- *
+ * @extends BaseOrderItemUnitRepository<T>
```

Comment:
Are both `@extends` and `@implements` are needed?

Reply:
Yup, if the extended class or implemented interface are generic, then we are obligated to define their type.